qa/rpc-tests/test_framework/mininode.py

```diff
@@ -606,20 +603,20 @@ def deserialize(self, f):
         self.nSequence = struct.unpack("<I", f.read(4))[0]
 
     def serialize(self):
-        r = ""
+        r = b""
         r += self.prevout.serialize()
         r += ser_string(self.scriptSig)
         r += struct.pack("<I", self.nSequence)
         return r
 
     def __repr__(self):
-        return "CTxIn(prevout=%r scriptSig=%s nSequence=%i)" \
```

Comment:
Is this change from `%r` to `%s` intended?

Reply:
I think I put that change in I can't remember the justification.   Based on [the meaning of "%r"](https://docs.python.org/3/library/stdtypes.html#printf-style-string-formatting)
 (a printable representation), I agree with @daira that changing to "%s" is incorrect.

Reply:
This seems not to have been resolved?